src/pantry.js

```diff
@@ -1,7 +1,7 @@
 import ingredientsData from "./data/ingredients";
 class Pantry {
-  constructor(userIngredients, user) {
-    this.contents = userIngredients;
+  constructor(user) {
+    this.contents = user.pantry;
```

Comment:
I changed the properties and the parameters to make more sense for the data its accessing. 
The userIngredients wasn't doing anything beneficial (currently as least).

Reply:
I like this change since user has access to the pantry